docs/faq.md

```diff
@@ -230,6 +230,50 @@ demonstration purposes).
 5. Re-authorizing the application with the new scope (GCM should automatically
 initiate this flow for you next time access is requested).
 
+### Q: What do the `configure` and `unconfigure` commands do?
+
+#### `configure`
+
+The `configure` command will set up Git to use GCM exclusively as the credential
+helper. The `configure` command is automatically called by the installers for
+Windows and macOS, but you can also run it manually.
+
+It will also set Git to provide the full remote URL (including path) to
+credential helpers for Azure Repos remotes using the `dev.azure.com` URL format.
+This is required in order to be to able to correctly identify the correct
+authority for that Azure DevOps organziation.
+
+Specifically, the `configure` command will modify your user Git configuration to
+include the following lines:
+
+```ini
+[credential]
+    helper =
+    helper = <path-to-gcm>
+[credential "https://dev.azure.com"]
+    useHttpPath = true
+```
+
+..where `<path-to-gcm>` is the absolute path to the GCM executable.
+
+The empty `helper =` line makes sure that existing credential helpers that may
+be set in the system Git configuration are not used.
```

Comment:
Perhaps tack on:

For more details, see the [`credential.helper` documentation](https://git-scm.com/docs/gitcredentials#Documentation/gitcredentials.txt-helper).